docusaurus/docs/iOS/02-tutorials/03-livestream.mdx

```diff
@@ -5,328 +5,325 @@ description: How to build a livestream experience using Stream's video SDKs
+For example, you can listen to this value with the `onReceive` modifier:
 
 ```swift
-.task {
-    for await videoEvent in call.subscribe() {
-        switch videoEvent {
-        case .typeCallBroadcastingStartedEvent:
-            self.isBroadcasting = true
-            self.isLoading = false
-        case .typeCallBroadcastingStoppedEvent:
-            self.isBroadcasting = false
-            self.isLoading = false
-        default:
-            break
-        }
+.onReceive(state.$ingress) { _ in
+    if let rtmp = state.ingress?.rtmp {
+        print("RTMP url and streamingKey \(rtmp)")
     }
 }
 ```
 
-When the call is started, there's still no livestream available. We need to explicitly start it, by calling the `startBroadcasting` method from the `call` object:
+B. Open OBS and go to settings -> stream
 
-```swift
-try await call.startBroadcasting()
-```
+- Select "custom" service
+- Server: equal to the server URL from the log
+- Stream key: equal to the stream key from the log
 
-When this method is called, it takes few seconds until the broadcasting is started. You can show a loading spinner until you receive the `.typeCallBroadcastingStartedEvent` event.
+Press start streaming in OBS. The RTMP stream will now show up in your call just like a regular video participant.
+Now that we've learned to publish using WebRTC or RTMP let's talk about watching the livestream.
 
-When the call is being broadcasted, we show a "Stop stream" button to the hosts, to allow them to stop broadcasting:
+### Step 5 - Viewing a livestream (WebRTC)
 
-```swift
-if isBroadcasting {
-    Button {
-        Task {
-            loading = true
-            try await call.stopBroadcasting()
-        }
-    } label: {
-        Text("Stop stream")
-    }
-} else {
-    Button {
-        Task {
-            loading = true
-            try await call.startBroadcasting()
-        }
-    } label: {
-        Text("Start stream")
-    }
-}
-```
+Watching a livestream is even easier than broadcasting.
 
-### Step 3 - Show the video using HLS
+Compared to the current code in in `LivestreamSampleApp.swift` you:
 
-There are two ways how the consumers can watch the video. The first one is via WebRTC, where you will join the call with the provided id, similarly to how it was created. The seconds one is via HLS streaming. 
+* Don't need to request permissions or enable the camera
+* Don't render the local video, but instead render the remote video
+* Typically include some small UI elements like viewer count, a button to mute etc.
 
-HLS is cheaper and has good buffering. However, it has a delay of around 15 seconds. WebRTC is ultra low latency, but has a higher cost and worse buffering.
+### Step 6 - (Optional) Viewing a livestream with HLS
```

Comment:
```suggestion
### Step 7 - (Optional) Viewing a livestream with HLS
```

Reply:
Or you can use 6.1 & 6.2 Step naming

Reply:
skipping to keep the same structure with Android tutorial